examples/communication/ip/ReceiveIPDataSample/ReadMe.txt

```diff
@@ -0,0 +1,89 @@
+  Introduction
+  ------------
+  This sample Java application shows how IP data messages are received from 
+  another XBee device connected to the Internet.
+
+  The application prints the received IP data to the standard output in ASCII 
+  and hexadecimal formats after the sender IP address.
+
+  NOTE: This example uses the Wi-Fi device (WiFiDevice) class, but it can be 
+        applied to other Internet capable XBee device classes such as 
+        CellularDevice.
+
+
+  Files
+  -----
+    * com.digi.xbee.api.receiveipdata.MainApp.java:
+      Main application class. It instantiates a Wi-Fi device and establishes a 
+      serial connection with it.
+
+    * com.digi.xbee.api.receiveipdata.MyIPDataReceiveListener.java:
+      Class that handles the received IP data messages.
+
+
+  Requirements
+  ------------
+  To run this example you will need:
+
+    * At least two Internet capable XBee radios (for example Wi-Fi radios) in 
+      API mode and their corresponding carrier boards (XBIB or equivalent).
+    * The XCTU application (available at www.digi.com/xctu).
+
+
+  Example setup
+  -------------
+    1) Plug the XBee radios into the XBee adapters and connect them to your
+       computer's USB or serial ports.
+
+    2) Ensure that the modules are in API mode and connected to the same access
+       point.
+       For further information on how to perform this task, read the 
+       'Configuring Your XBee Modules' topic of the Getting Started guide.
+
+    3) Set the port and baud rate of the receiver XBee radio in the MainApp 
+       class.
+       If you configured the modules in the previous step with the XCTU, you 
+       will see the port number and baud rate in the 'Port' label of the device 
+       on the left view.
+
+
+  Running the example
+  -------------------
+  First, build and launch the application. Then, you need to send a data frame 
+  to the receiver (local) module from another device. Follow the steps below to 
+  do so:
+
+    1) Launch the XCTU application.
+
+    2) Add the sender (remote) XBee module to the XCTU, specifying its port 
+       settings.
+
+    3) Once the module is added, change to the 'Consoles' working mode and 
+       open the serial connection.
+
+    4) Create and add a frame using the 'Frames Generator' tool with the 
+       following parameters:
+
+       - Protocol:                        Select the protocol of your device.
+       - Frame type:                      0x20 - TX IPv4
+       - Frame ID:                        01
+       - IPv4 32-bit dest. address:       The IP address ('MY') of the receiver module
+                                          in hexadecimal format.
+       - 16-bit dest. port:               The port number ('C0') of the receiver module,
+                                          26 16 by default.
+       - 16-bit source port:              00 00
+       - Protocol:                        What you prefer, TCP, UDP or TCP SSL.
```

Comment:
I would remove TCP SSL from here, it is only available for Cellular and we have not tested it with the library,